src/styles/ui/CardButton.scss

```diff
@@ -2,11 +2,20 @@
 
 .card {
   &:hover {
-    transform: translateY(-2em);
+    transform: translateY(-1.5em);
   }
   &.whiteCard {
     background-color: $WHITE_CARD_BACKGROUND_COLOR;
     color: $WHITE_CARD_TEXT_COLOR;
+
+    .scrollable-div {
+      overflow-y: auto;
+      height: 240px;
+      word-break: break-word;
+      scrollbar-width: thin;
```

Comment:
There is no scrollbar-width.
If you want to edit the scrollbar, do like I did in chat. E.g
![image](https://user-images.githubusercontent.com/75019468/169707926-31a9b019-330b-4238-8add-6bf66d7770a7.png)
Can you add some padding-right so that the text does not touch the scrollbar?

Reply:
done (fontsize 20 for test):
![grafik](https://user-images.githubusercontent.com/91314660/169709814-342d2792-5c34-496b-92c5-f91a2c53bfbf.png)